component/src/main/java/me/androidbox/component/agenda/AgendaCardType.kt

```diff
@@ -0,0 +1,170 @@
+package me.androidbox.component.agenda
+
+import android.util.Log
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import me.androidbox.component.R
+import me.androidbox.component.general.OptionButton
+import me.androidbox.component.ui.theme.*
+
+@Composable
+fun AgendaCard(
+    modifier: Modifier = Modifier,
+    agendaCardType: AgendaCardType,
+    isAgendaCompleted: Boolean = false,
+    title: String,
+    subtitle: String,
+    dateTimeInfo: String,
+    onMenuOptionClicked: () -> Unit
+) {
```

Comment:
This is kinda messy
Since the first parameters should be parameters that don't have a default value, and then you have parameters with default value

So something like:

```
title: String,
subtitle: String,
dateTimeInfo: String,
modifier: Modifier = Modifier,
agendaCardType: AgendaCardType,
isAgendaCompleted: Boolean = false,
onMenuOptionClicked: () -> Unit
```

The only exception, is if you want to have a lambda like onMenuOptionClicked, which would be at the end, just like you did